protocols/bftcosi/bftcosi.go

```diff
@@ -61,18 +62,11 @@ type ProtocolBFTCoSi struct {
 	// channel for challenge during the commit phase
 	challengeCommitChan chan challengeCommitChan
 	// channel for response
-	responseChan chan responseChan
+	responseChan chan []responseChan
```

Comment:
Ah, I remember. Maybe we should change the SDA-way then. But that's not related to this issue.